fenix/app/src/main/java/org/mozilla/fenix/settings/search/SearchEngineFragment.kt

```diff
@@ -44,6 +47,9 @@ class SearchEngineFragment : PreferenceFragmentCompat() {
         requirePreference<SwitchPreference>(R.string.pref_key_show_nonsponsored_suggestions).apply {
             isVisible = context.settings().enableFxSuggest
         }
+        requirePreference<SwitchPreference>(R.string.pref_key_learn_about_fx_suggest).apply {
```

Comment:
```suggestion
        requirePreference<Preference>(R.string.pref_key_learn_about_fx_suggest).apply {
```

I think we'll crash with a bad cast exception if the type passed to `requirePreference` doesn't match the type of the element in the XML.

Reply:
ooops